Include/httpClient/async.h

```diff
@@ -71,7 +65,7 @@ typedef struct AsyncBlock
     /// <summary>
     /// Internal use only
     /// </summary>
-    unsigned char internal[ASYNCBLOCK_INTERNAL_SIZE];
+    unsigned char internal[sizeof(void*) * 4];
```

Comment:
why 4?

Reply:
I'm a sucker for powers of two.  We won't be able to change this header once published and I wanted a bit of headroom.